core/common/src/grpc_interceptor_samples.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+use bytes::Bytes;
+use core_protobuf_data_access::invehicle_digital_twin;
+use log::info;
+use prost::Message;
+use std::error::Error;
+
+use crate::grpc_interceptor::GrpcInterceptor;
+
+/// Sample gRPC interceptor.
+#[derive(Clone)]
+pub struct SampleGrpcInterceptor {}
```

Comment:
can you document more about this struct? either here or on the method implementations? It's not immediately clear how this struct modifies the data it's intercepting

Reply:
I provided a README.  More will be added to it later.